packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts

```diff
@@ -101,38 +98,26 @@ const getOrganizedParticipants = (props: OrganizedParticipantsArgs): OrganizedPa
   return { gridParticipants, overflowGalleryParticipants };
 };
 
-interface SortedRemoteParticipants {
-  [key: string]: VideoGalleryRemoteParticipant;
-}
-
 /**
  * Hook to determine which participants should be in grid and overflow gallery and their order respectively
  * @private
  */
 export const useOrganizedParticipants = (props: OrganizedParticipantsArgs): OrganizedParticipantsResult => {
-  // map remote participants by userId
-  const remoteParticipantMap = props.remoteParticipants.reduce((map, remoteParticipant) => {
-    map[remoteParticipant.userId] = remoteParticipant;
-    return map;
-  }, {} as SortedRemoteParticipants);
-
   const spotlightedParticipantUserIds = props.spotlightedParticipantUserIds ?? [];
   const pinnedParticipantUserIds = props.pinnedParticipantUserIds ?? [];
-  // declare set of focused participant user ids as spotlighted participants user ids followed by
-  // pinned participants user ids which is deduplicated while maintaining order
-  const focusedParticipantUserIdSet = new Set(
-    spotlightedParticipantUserIds.concat(pinnedParticipantUserIds).filter((p) => remoteParticipantMap[p])
-  );
-  // get focused participants from map of remote participants in the order of the user ids
```

Comment:
@mgamis-msft please check the new logic conforms to this. I am not sure about `in the order of the user ids` as I don't see any sorting by userId happening

Reply:
Looks good to me! Thanks for the cleanup